doc/whats_new.rst

```diff
@@ -214,6 +214,8 @@ Changelog
 
 - Add ``mne.realtime.MockLSLStream`` to simulate an LSL stream for testing and examples by `Teon Brooks`_
 
+- Add support for file-like objects in :func:`mne.epochs.read_epochs` as long as preloading is used by `Paul Roujansky`_
```

Comment:
Can you add a link at the bottom? It's probably why CricleCI complains

Reply:
My bad, thanks for that @agramfort